NickvisionTubeConverter.Shared/Resources/Strings.resx

```diff
@@ -225,7 +225,7 @@
 	</data>
 
 	<data name="NoDownloads.Description" xml:space="preserve">
-		<value>Add a download to get started.</value>
+		<value>Add a video URL to start downloading.</value>
```

Comment:
How about "Add a video, audio or playlist URL to start downloading."?

Reply:
That works :)